packages/wxt/e2e/tests/npm-packages.test.ts

```diff
@@ -21,7 +21,8 @@ test('Only one version of esbuild should be installed (each version is ~20mb of
     if (name === 'esbuild') esbuildVersions.add(meta.version);
   });
 
-  expect([...esbuildVersions]).toHaveLength(1);
+  // TODO: Revert to 1 once vite is upgraded to >6.0.8
+  expect([...esbuildVersions]).toHaveLength(2);
```

Comment:
Currently, tsx only support esbuild 0.23 or 0.25, while vite 6.0.8 supports 0.24... So for now, there's two versions, until I can update vite to a newer version that supports 0.25 as well.